churaverse-plugins-client/src/synchroBreakPlugin/synchroBreakPlugin.ts

```diff
@@ -128,6 +128,17 @@ export class SynchroBreakPlugin extends CoreGamePlugin {
    * シンクロブレイク特有の開始時に実行される処理
    */
   protected handleGameStart(): void {
+    // ゲーム専用の中断文言セット
+    this.gamePluginStore.gameAbortAlertConfirm.setMessage('シンクロブレイクを中断しますか？')
```

Comment:
アラート文言の切り替えがゲームごとに必要になると思うので、共通の関数を作ってもいいと思いました。

```suggestion
    this.gamePluginStore.gameAbortAlertConfirm.showGameAbortMessage(this.gameName)
```

Reply:
gameNameのみで、文言を設定できるように修正しました